assets/js/scenes.js

```diff
@@ -43,6 +44,33 @@ const generateScenes = () => {
 
     // spawn baby yoda
     spawnBabyYoda();
+    
+    // display score
+    add([
+        text("Score:0"),
+        pos(10, 0),
```

Comment:
Suggestion: You could use the `width()` and `height()` methods available to return the canvas' width and height. You will then be able to place it based on those. That way, if we were to ever resize the canvas, the text would be positioned proportionally.